internal/querycoord/task.go

```diff
@@ -72,16 +72,15 @@ type task interface {
 	msgBase() *commonpb.MsgBase
 	msgType() commonpb.MsgType
 	timestamp() Timestamp
-	getTriggerCondition() querypb.TriggerCondition
-	setTriggerCondition(trigger querypb.TriggerCondition)
+	getPriority() int
+	setPriority(priority int)
```

Comment:
We need comment to claim that setPriority() for added task won't change the priority of processing it of the task scheduler